web/src/test/manual/web/kmpHeader.js

```diff
@@ -0,0 +1,89 @@
+
+/*
+  This script allows its host page to add a keyboard via KMP file.
+
+  Expectations for the host page:
+  - `<p> <input type="file" id="package-file-select" accept=".kmp"/> </p>`
+  - <script src="../../../../build/tools/testing/bulk_rendering/lib/zip.min.js" type="application/javascript"></script>
+*/
+
+window.addEventListener('load', () => {
```

Comment:
This is the same code we have in the bulk-renderer's "local-renderer.html" page for loading the keyboard from its KMP.  We don't need the other parts of that here, though.

Reply:
See #10432.